frontend/src/layout/lemonade/Breadcrumbs/Breadcrumbs.scss

```diff
@@ -27,9 +27,9 @@
     font-weight: $medium;
     white-space: pre;
     user-select: none;
+    padding-bottom: 4px; // For popovers to have some room
```

Comment:
I did the same thing in https://github.com/PostHog/posthog/pull/7081 but in `.Popup` styling so that buttons in general have this breathing room, so you can (and should 😅 ) remove this and just merge with `master`

Reply:
Perfect!